internal/delete/delete_item_action_handler.go

```diff
@@ -62,16 +62,17 @@ func InvokeDeleteActions(ctx *Context) error {
 	dir, err := archive.NewExtractor(ctx.Log, ctx.Filesystem).UnzipAndExtractBackup(ctx.BackupReader)
 	if err != nil {
 		return errors.Wrapf(err, "error extracting backup")
-
 	}
 	defer ctx.Filesystem.RemoveAll(dir)
 	ctx.Log.Debugf("Downloaded and extracted the backup file to: %s", dir)
 
 	backupResources, err := archive.NewParser(ctx.Log, ctx.Filesystem).Parse(dir)
-	if err != nil {
+	if existErr := errors.Is(err, archive.ErrNotExist); err != nil && !existErr {
 		return errors.Wrapf(err, "error parsing backup %q", dir)
+	} else if existErr {
+		ctx.Log.Debug("ignore deleting resources: ", err)
```

Comment:
1.  Why not write it like 
```
if existErr := errors.Is(err, archive.ErrNotExist); existErr {
.....
} else if err != nil {
...
}
```

2. I think the log message is misleading, here you actually ignore invoking the delete item actions, not ignore deleting resources....

Reply:
@reasonerjt I agree, I added your suggestion.

Reply:
thanks!